src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -286,12 +286,12 @@ export default {
 		 */
 		getWidth(el) {
 			if (!el.classList) return 0
-			const hide = el.classList.contains('crumb--hidden')
+			const hide = el.classList.contains('vue-crumb--hidden')
```

Comment:
Maybe we should use a constant to store the `vue-crumb` class prefix :wink:

Reply:
Would be good. I put it on the list.